aio/content/examples/getting-started/src/app/shipping/shipping.component.ts

```diff
@@ -11,15 +12,18 @@ import { CartService } from '../cart.service';
   styleUrls: ['./shipping.component.css']
 })
 // #docregion props
-export class ShippingComponent {
+export class ShippingComponent implements OnInit {
 
-  shippingCosts = this.cartService.getShippingPrices();
+  shippingCosts: Observable<{ type: string, price: number }[]>;
```

Comment:
Actually, this change causes TypeScript to complain (and CI to fail) with:

```
Property 'shippingCosts' has no initializer and is not definitely assigned in the constructor
```

Maybe use:

```suggestion
  shippingCosts!: Observable<{ type: string, price: number }[]>;
```

Reply:
Hii @gkalpak , Thanks for the suggestions, I have done your suggested changes. If any other changes are required let me know.

Thanks